backend/dal/Services/UserService.cs

```diff
@@ -80,10 +80,10 @@ public User Activate()
                 this.Logger.LogInformation($"User Activation: key:{key}, email:{email}, username:{username}, first:{givenName}, surname:{surname}");
 
                 var person = new Person(surname, givenName);
-                var contactMethodType = this.Context.ContactMethodTypes.First(cmt => cmt.Id == "EMAIL");
-                var contactMethod = new ContactMethod(person, organization, contactMethodType, email);
+                var contactMethod = new ContactMethod(person, organization, ContactMethodTypes.WorkEmail, email);
```

Comment:
Even though I could revert this since I'm adding the previous constructor back, this is still more concise.